### PR TITLE
Set up sportclass schema and relationship

### PR DESCRIPTION
**Pull request by ZsokaM**

### CODE REVIEW

File: schemas/SportClass.ts [hunk 1]
```diff
@@ -26,6 +21,23 @@ export const SportClass = list({
...
+    day: select({
+      dataType: 'enum',
+      options: [
+        { label: 'Monday', value: 'Monday' },
+        { label: 'Tuesday', value: 'Tuesday' },
+        { label: 'Wednesday', value: 'Wednesday' },
+        { label: 'Thursday', value: 'Thursday' },
+        { label: 'Friday', value: 'Friday' },
+        { label: 'Saturday', value: 'Saturday' },
+        { label: 'Sunday', value: 'Sunday' },
```
Comment (michaelkoek) [inline schemas/SportClass.ts]: Why is this an object if the two properties share the same value?
This could be an array

Comment (michaelkoek) [reply schemas/SportClass.ts]: I'm also wondering why the backend needs to know which day it is. Why should it care if it's on a Monday, Wednesday or Friday? The backend should be interested in the the date- or timestamp the class is picked.
Or is there a reason why you need to have this?

Comment (ZsokaM) [reply schemas/SportClass.ts]: Keystone specifies a certain set of fields you can use in schemas. 'Select' is one of them, where the label is a string to be displayed in the Admin UI and the value will be used in the GraphQL API and stored in the database. The Admin UI can be used to create documents (I was originally thinking about skipping this in the frontend all together but then the project doesnt conform with the CRUD requirements). I tested how it looks like if I use only a text input without strict specs and found it nicer to have drops downs to select from.

File: schemas/SportClass.ts [hunk 2]
```diff
@@ -1,51 +1,49 @@
...
+        { label: 'Wednesday', value: '2' },
+        { label: 'Thursday', value: '3' },
+        { label: 'Friday', value: '4' },
+        { label: 'Saturday', value: '5' },
+        { label: 'Sunday', value: '6' },
       ],
       isRequired: true,
       ui: { displayMode: 'select' },
     }),
+    startTime: text({
```
Comment (ZsokaM) [inline schemas/SportClass.ts:44]: no float number support so i went with text

**Pull request merged**